package-lock.json

```diff
@@ -0,0 +1,9817 @@
+{
+  "name": "yamui",
```

Comment:
did you switch to yarn to npm? do we want to manage both yarn.lock and package-lock.json?

Reply:
Sorry, no, I shouldn't have checked this in. This exploratory branch is a mess! Cleaning up now and will be ready for review this am.